Documentation/gettingstarted/kind.rst

```diff
@@ -0,0 +1,224 @@
+
+.. code:: bash
+
+    kind create cluster --config=kind-config.yaml
+
+This will add a ``kind-kind`` context to ``KUBECONFIG`` or if unset ``${HOME}/.kube/config``
+
+.. code:: bash
+
+    kubectl cluster-info --context kind-kind
+
+.. _kind_install_cilium:
+
+Install Cilium
+==============
+
+.. include:: k8s-install-download-release.rst
+
+.. note::
+
+    As Kind is running nodes as containers in Docker, they're sharing your host machines' kernel.
+    So we have to disable :ref:`host-services` otherwise kind clusters will clobber each other
+    and/or Cilium won't be able to restart until the host is restarted.
+
+Install Cilium release via Helm:
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.nodeinit.enabled=true \\
+      --set global.kubeProxyReplacement=partial \\
+      --set global.externalIPs.enabled=true \\
+      --set global.nodePort.enabled=true \\
+      --set global.hostPort.enabled=true
+
+.. include:: k8s-install-validate.rst
+.. include:: hubble-install.rst
+
+Next steps
+==========
+
+Now that you have a Kubernetes cluster with Cilium up and running, you can take
+a couple of next steps to explore various capabilities:
+
+* :ref:`gs_http`
+* :ref:`gs_dns`
+* :ref:`gs_cassandra`
+* :ref:`gs_kafka`
+
+Tips
+====
+
+Pre-load load Cilium images into the kind cluster so each worker doesn't have to pull them.
+
+Run this before installing Cilium:
+
+.. parsed-literal::
+
+  docker pull cilium/cilium:v|CHART_RELEASE|
+  kind load docker-image cilium/cilium:v|CHART_RELEASE|
+
+.. _gs_kind_cluster_mesh:
+
+Custer Mesh
+===========
+
+With Kind we can simulate Cluster Mesh in a sandbox too.
+
+Kind Configuration
+------------------
+
+This time we need to create (2) ``config.yaml``, one for each kubernetes cluster.
+We will explicitly configure their ``pod-network-cidr`` and ``service-cidr`` to not overlap.
+
+Example ``kind-cluster1.yaml``:
+
+.. code:: yaml
+
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    networking:
+      disableDefaultCNI: true
+      podSubnet: 10.0.0.0/16
+      serviceSubnet: 10.1.0.0/16
+
+Example ``kind-cluster2.yaml``:
+
+.. code:: yaml
+
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    networking:
+      disableDefaultCNI: true
+      podSubnet: 10.2.0.0/16
+      serviceSubnet: 10.3.0.0/16
+
+Create Kind Clusters
+--------------------
+
+We can now create the respective clusters:
+
+.. code:: bash
+
+    kind create cluster --name=cluster1 --config=kind-cluster1.yaml
+    kind create cluster --name=cluster2 --config=kind-cluster2.yaml
+
+Deploy Cilium
+-------------
+
+This is the same helm command as from :ref:`_kind_install_cilium` however
```

Comment:
The docs build complained:

```
Validating documentation (syntax, spelling)...

Please fix the following documentation warnings:
/github/workspace/Documentation/gettingstarted/kind.rst:174: WARNING: undefined label: _kind_install_cilium (if the link has no caption the label must precede a section header)
```

I believe that for the reference here you need `kind_install_cilium`, not `_kind_install_cilium`.

I believe that you should be able to validate things like this using `make -C Documentation` in your local development environment, but you're also welcome to push the PR and have the github workflow at the bottom of this PR provide you this feedback.

Reply:
Doh, sorry. Force of habit right now. I'm on severely limited bandwidth so my normal routine is just let CI run tests. I'll use that locally since I assume it won't be downloading that much in dependencies :)